docs/build/guides/dapps/soroban-contract-init-template.mdx

```diff
@@ -29,11 +29,11 @@ The official template maintained by Stellar Development Foundation (SDF), as use
 
 (You may wonder why it makes this unpopular choice. A fair question! The team wanted to balance actual utility with broad approachability. Not everyone learning Stellar and Soroban is familiar with React, or any other UI library. It also demonstrates that core Soroban libraries all work with any JavaScript project.)
 
-But note that the CLI's `contract init` command was made to work with _any_ frontend template. It need not even live on GitHub! Simply pass another valid git URL to the command, and it will clone that instead of the official repository:
+But note that the CLI's `contract init` command was made to work with a specific set of templates. To use other templates, we will clone other templates from repository:
```

Comment:
```suggestion
To use other templates, we will clone them from their repositories:
```

Reply:
This change just removes the `contract init` reference, since the fe templates are independent of init now.

Reply:
Yes, fixed for diction